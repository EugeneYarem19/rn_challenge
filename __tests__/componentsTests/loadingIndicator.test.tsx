import React from "react";

import { LoadingIndicator, } from "@components";

import { snapshotTest, } from "@tests/__utils__";

describe("LoadingIndicator tests", () => {
  test("LoadingIndicator must renders correctly", () => {
    snapshotTest(<LoadingIndicator />);
  });
});
