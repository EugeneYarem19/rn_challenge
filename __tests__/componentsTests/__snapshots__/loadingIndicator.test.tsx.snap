// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator tests LoadingIndicator must renders correctly 1`] = `
<ActivityIndicator
  animating={true}
  color="#e59f2b"
  hidesWhenStopped={true}
  size="large"
/>
`;
